test/source/TestString.inl

```diff
@@ -1163,7 +1163,7 @@ int TEST_STRING_NAME()
 		typename StringType::value_type buf[64];
 
 		StringType str(LITERAL("abcdefghijklmnopqrstuvwxyz"));
-		str.copy(buf, 10, sizeof buf);
+		str.copy(buf, 10, 0);
```

Comment:
I think this is still wrong.  
str.copy(buf, 10, 10);  // std::basic_string(ptr, size, position)